apps/nextjs-website/next.config.js

```diff
@@ -1,4 +1,4 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {}
 
-module.exports = nextConfig
+module.exports = nextConfig
```

Comment:
To avoid committing these types of changes, configure Git properly to handle line endings[^1].

[^1]: https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings?platform=windows